titanic/titanicML.ipynb

```diff
@@ -225,7 +224,7 @@
    "outputs": [],
    "source": [
     "# Initialize our algorithm\n",
-    "lr = LogisticRegression(random_state=1)"
+    "lr = LogisticRegression(random_state=1, solver='liblinear')"
```

Comment:
Explicitly set `solver='liblinear'` in order to silence the `FutureWarning` from scikit-learn about the default parameter value changing from `liblinear` to `lbfgs` in 0.22.